### DISCUSSION TITLE
CultureInfo class not loading native digits

### DISCUSSION THREAD

**Discussion by raihaniqbal** [xamarin/Xamarin.Forms#14110] [Q&A]
I'm trying to localize my Xamarin forms app and I noticed that my datetimes were not being localized. When I inspected the culture info object, I noticed the NativeDigits array was populated with English digits.

This is what I get in my Xamarin project
![Xamarin project culture info object](https://i.stack.imgur.com/oPw8B.png)

This is what it should look like
![Screenshot from .NET forms project](https://i.stack.imgur.com/RSw3v.png)

I have sample application which demonstrates this.
https://github.com/raihaniqbal/MultilingualXFSample

Fyi, this issue is not restricted to Arabic. You can pick any other language that has non-English digits.

Just add the following code in a XF project and inspect the CultureInfo object:
`CultureInfo.GetCultureInfo("ar-AE")`

**Comment by EmilAlipiev** [comment]
Please, ask your question here and tag Xamarin.Forms and Xamarin https://docs.microsoft.com/en-us/answers/products/dotnet?WT.mc_id=xamarinforumsredirect-web-xamarin

**Reply by raihaniqbal** [reply]
It's been done already.

https://docs.microsoft.com/en-us/answers/questions/340093/xamarin-forms-project-getting-incorrect-native-dig.html?childToView=348989#comment-348989

Someone from MSFT over there suggested I should open a ticket which I've done as well.

https://github.com/xamarin/Xamarin.Forms/issues/14128
